apps/api/src/server/app/validation/validation.controller.js

```diff
@@ -86,4 +88,14 @@ async function getAppealForValidation(appealId) {
 	return appeal;
 }
 
-export { getAppeals, getAppealDetails, updateAppeal, submitValidationDecision };
+const obtainLPAList = async function() {
+	const dataRaw = await got.get('https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LPA_APR_2021_UK_NC/FeatureServer/0/query?where=1%3D1&outFields=LPA21NM&outSR=4326&f=json');
```

Comment:
What happen if this request fails?

Reply:
500 with the error message of failure, can add to test